contribution/zeppelinweb/GPG01.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: sideMenu
```

Comment:
File name is a bit confusing here as it's not related to GPG aka GnuPG https://www.gnupg.org/